db/seeds.rb

```diff
@@ -1,7 +1,33 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create(email: 'user@gmail.com', username: 'user', password: 'user11', password_confirmation: 'user11')
+
+type1 = Type.create(name: 'Warrior', user_id: 1)
+type1.image.attach(io: File.open("#{Rails.root}/app/assets/images/warrior.png"), filename: "warrior.png")
```

Comment:
@VanjaJeftic all fixtures required for testing purposes should be in one of the testing dirs. Under /app/assets we should only have production things.

Read this https://axiomq.com/blog/rspec-and-factorygirl-setup-for-testing-carrierwave-uploaders/ where we put files in `/spec/fixtures/myfiles` (or any other more meaningful dir name).

Oh... I just realized this is for seed. In that case it should be under `/db/seeds/type_images` or similar.